lisa/tools/dmesg.py

```diff
@@ -69,7 +69,12 @@ def get_vmbus_version(self) -> VersionInfo:
         result.assert_exit_code(
             message=f"exit code should be zero, but actually {result.exit_code}"
         )
-        raw_vmbus_version = re.finditer(self.__vmbus_version_pattern, result.stdout)
+
+        # Remove the color code from stdout stream
+        color_code_pattern = r"\x1b\[[0-9;]*m"
```

Comment:
can you paste a sample lin in the comments?

Reply:
Hi Lili, i cannot paste the sample in comment as it is related to color code. dmesg stdout stream will have it. Here is the screenshot. VMBus version sample lines are already there in the code.

![image](https://github.com/microsoft/lisa/assets/113590758/e074d2da-ac2a-438c-b7fe-f63ed7368ef4)